Gems/PhysX/Code/Source/Scene/PhysXScene.cpp

```diff
@@ -151,8 +150,8 @@ namespace PhysX
                                 shapeAdded = true;
```

Comment:
wondering if AddShape should return `true` if **all** shapes from the list got created, not if at least one is successful.
opinions?